modules/widgets/eu-cookie-law/widget-amp.php

```diff
@@ -26,11 +26,11 @@
 				echo esc_html( $instance['customtext'] );
 			}
 
-			$policy_link_text = 'default' === $instance['policy-url'] || empty( $instance['custom-policy-url'] )
-					? $instance['default-policy-url']
-					: $instance['custom-policy-url'];
+			$is_default_policy = 'default' === $instance['policy-url'] || empty( $instance['custom-policy-url'] );
+			$policy_link_url   = $is_default_policy ? $instance['default-policy-url'] : $instance['custom-policy-url'];
+			$policy_link_rel   = $is_default_policy ? 'nofollow' : '';
 			?>
-			<a href="<?php echo esc_url( $policy_link_text ); ?>">
+			<a href="<?php echo esc_url( $policy_link_url ); ?>" rel="<?php echo esc_attr( $policy_link_rel ); ?>">
```

Comment:
I would be tempted to not add `rel=""` at all when we're using a custom policy. What do you think?

Reply:
The default value of rel is empty and AFAIK it's not an issue for it to be defined explicitly and still be empty. The alternative is to output the entire attribute and then disable the "no output without an escaping function" lint in the middle of html output e.g.:
```html
<a
    href="<?php echo esc_url( $policy_link_url ); ?>"
    <?php echo esc_attr( $policy_link_rel ); ?> <?php // phpcs:ignore WordPress.XSS.EscapeOutput.OutputNotEscaped ?>
>
```
 I'm fine with either - just decided to go the other route to reduce the noise in the code. Let me know.

Reply:
Yeah that's a good point. Good to go then! 👍